### ISSUE TITLE
Feature-Request: Skip Bad Serials When Importing From File

### ISSUE THREAD

**Issue by somefunguy** [apocalyptech/bl3-cli-saveedit#25]
when importing items from file, it would be nice to skip and announce error for a bad bl3() code rather than stopping the import altogether.

**Comment by apocalyptech** [comment]
Yep, agreed!  Will hopefully get to this over the weekend.  Cheers!
